### ISSUE TITLE
Save previously used user/window manager

### ISSUE THREAD

**Issue by ProspectPyxis** [JezerM/nody-greeter#36]
## Environment
- OS: Arch Linux
- nody-greeter version: 1.5.1

## Description of feature
In most greeters (in particular, `lightdm-gtk-greeter`), the last user that logged in, and the window manager that user used, is saved for the next login. However, this greeter, by default, will always select the first user alphabetically, and the first window manager alphabetically. While this behavior can be changed directly by a theme (for example, the Aether theme has a configuration option for the default user), I'd like this to be the standard behavior, as I believe that's much more convenient for most users.

As a side note, for saving the window manager, LightDM is supposed to write a `~/.dmrc` file that contains the window manager last selected - however, this file isn't created at all when using this greeter, and I'm assuming it's never read, either.

## Possible solutions
The workaround I've been using, at least for default window managers, is directly editing the files in `/usr/share/xsessions` and adding a number in front to force the window manager I want to be the first alphabetically. However, I don't think a workaround like this exists for users.

**Comment by JezerM** [comment]
Sorry for the late response.

> As a side note, for saving the window manager, LightDM is supposed to write a ~/.dmrc... however, this file isn't created at all when using this greeter, and I'm assuming it's never read, either.

Yep, however web-greeter/nody-greeter/sea-greeter can't read neither write in the home directory, so the `.dmrc` file is exclusively handled by lightdm. Not even `lightdm-gtk-greeter` handles this.

> In most greeters, the last user that logged in, and the window manager that user used, is saved for the next login.

This should be the default behavior, at least for **gruvbox** and **dracula** themes. Which theme are you using?

**Comment by ProspectPyxis** [comment]
Huh, the issue seems to have resolved itself on its own for me. I believe it could've been a dependency issue, since I updated my system around the same time, but I'm not sure on that.

I tried the default gruvbox and dracula themes, and I also tried your fork of the Aether theme. I now use your lightdm-void-theme, but even when switching back to the other themes, the issue seems to be gone.

Closing this since I can no longer reproduce.
